data/mods/obsolete-mods.json

```diff
@@ -1 +1 @@
-[ "ags", "Medieval_pack", "RemoteVehicles", "RemoteControls_pack", "old_guns", "no_magazines" ]
+[ "ags", "Medieval_pack", "RemoteVehicles", "RemoteControls_pack", "old_guns", "no_magazines", "No_Diamond_Weapons" ]
```

Comment:
I did look at that option but maintaining a mod to adjust the lab finales by 12% seems a bit expensive in terms of maintenance? I'm presuming the mod was in reaction to recipe bloat so I can't see any other reason to preserve it's existance?